BuildingSync.xsd

```diff
@@ -4076,6 +4076,78 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="BuildingEnergyUse" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The annual amount of all the energy the building consumes onsite. Calculated as imported energy + onsite renewable energy - exported energy - net increase in stored energy (per ASHRAE 105-2014 Figure 5.6). (kBtu)</xs:documentation>
```

Comment:
Same as above:
`Calculated as imported energy (Eimp) + onsite renewable energy (Eg) - exported energy (Eexp) - net increase in stored imported energy (Es)`...